pkg/controller/controller.go

```diff
@@ -545,19 +552,19 @@ func (ctrl *resizeController) markPVCResizeFinished(
 	return nil
 }
 
-func (ctrl *resizeController) patchClaim(oldPVC, newPVC *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaim, error) {
-	patchBytes, err := util.GetPVCPatchData(oldPVC, newPVC)
+func (ctrl *resizeController) patchClaim(oldPVC, newPVC *v1.PersistentVolumeClaim, addResourceVersionCheck bool) (*v1.PersistentVolumeClaim, error) {
```

Comment:
could you add some comments about this option addResourceVersionCheck?

Reply:
Fixed.